src/MongoDB.Driver.Core/Core/Authentication/ScramShaAuthenticator.cs

```diff
@@ -14,12 +14,12 @@
 */
 
 using System;
-using System.Collections.Generic;
-using System.IO;
 using System.Security.Cryptography;
 using System.Text;
 using MongoDB.Bson.IO;
+#if NET452
```

Comment:
This is really confused when you see that some NSs are not used, but when you remove them from using section, your application cannot be built anymore.

Reply:
Ack